packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -780,6 +785,51 @@ test.serial('STORY05(cont): LPs withdraw all liquidity', async t => {
   t.truthy(b);
 });
 
+test.serial('withdraw all liquidity while ADVANCING', async t => {
+  const {
+    bridges: { snapshot, since },
+    common: {
+      commonPrivateArgs: { feeConfig },
+      utils,
+      brands: { usdc },
+      bootstrap: { storage },
+    },
+    evm: { cctp, txPub },
+    mint,
+    startKit: { zoe, instance, metricsSub },
+  } = t.context;
+
+  const usdcPurse = purseOf(usdc.issuer, utils);
+  // 1. Alice deposits 10 USDC for 10 FastLP
+  const alice = makeLP('Alice', usdcPurse(10_000_000n), zoe, instance);
+  await E(alice).deposit(t, 10_000_000n);
+
+  // 2. Bob initiates an advance of 6, reducing the pool to 4
+  const bob = makeCustomer('Bob', cctp, txPub.publisher, feeConfig);
+  const bridgePos = snapshot();
+  const sent = await bob.sendFast(t, 6_000_000n, 'osmo123bob5');
+  await eventLoopIteration();
+  bob.checkSent(t, since(bridgePos));
+
+  // 3. Alice proposes to withdraw 7 USDC
+  await t.throwsAsync(E(alice).withdraw(t, 0.7), {
+    message: /cannot withdraw .* in use; stand by/,
```

Comment:
Related to my comment above, possibly make this expected string more explicit

Reply:
OK.

I'm generally reluctant to commit to specific error messages in our API, but don't suppose I have a solid justification.